index.bs

```diff
@@ -1095,25 +1095,25 @@ enum MLOperandDataType {
 
 dictionary MLOperandDescriptor {
   required MLOperandDataType dataType;
-  sequence<[EnforceRange] unsigned long> dimensions = [];
+  sequence<[EnforceRange] unsigned long> shape = [];
 };
 </script>
 
 <dl dfn-type=dict-member dfn-for=MLOperandDescriptor>
     : <dfn>dataType</dfn>
     :: The operand data type.
 
-    : <dfn>dimensions</dfn>
+    : <dfn>shape</dfn>
     :: The shape of the operand. It is empty for scalar operands, and non-empty for tensor operands.
```

Comment:
Can we define "shape" more clearly rather than a self-referential definition (e.g. "a cat is a cat")? Maybe:

```
    : <dfn>shape</dfn>
    :: The list of dimensions for the operand. It is empty for scalar operands, and non-empty for tensor operands.
```
or
```
    : <dfn>shape</dfn>
    :: The list of dimension sizes for the operand. It is empty for scalar operands, and non-empty for tensor operands.
```

Also existing, but it's funky that we say "non-empty for tensor operands" even though scalars *are* tensor operands, just zero order operands. Maybe just "It is empty for scalar tensors.".

Reply:
Ack. Updated to:
```
:: The list of dimensions of the operand. It is empty for scalar operands.
```